dali/aux/optical_flow/optical_flow_adapter.h

```diff
@@ -36,16 +36,16 @@ struct compute_to_storage<kernels::ComputeGPU> {
 
 }  // namespace detail
 
-enum VectorGridSize {
+enum struct VectorGridSize {
```

Comment:
Why do we need `struct` here?

Reply:
So that it's not global. AFAIK, `enum struct` is just a scoped `enum`

Reply:
Right